test/unified_format.py

```diff
@@ -635,6 +635,7 @@ class UnifiedSpecTestMixinV1(IntegrationTest):
     a class attribute ``TEST_SPEC``.
     """
     SCHEMA_VERSION = Version.from_string('1.5')
+    RUN_ON_LOAD_BALANCER = True
```

Comment:
Is it certain that all Unified format tests added in the future will need to run against LoadBalanced topologies? If not, we might have a problem when some unified format tests have to not be run against LB topologies with this pattern. One way to keep this configurable would be to add a kwarg to `generate_test_classes` that allows us to set `RUN_ON_LOAD_BALANCER=False` when desired.

Reply:
> Is it certain that all Unified format tests added in the future will need to run against LoadBalanced topologies?
Yes, see https://github.com/mongodb/specifications/tree/master/source/load-balancers/tests#tests.